qcodes/station.py

```diff
@@ -748,10 +754,12 @@ def instrument_names_from_module(module: ModuleType) -> Tuple[str, ...]:
         submodules = list(pkgutil.walk_packages(module.__path__, module.__name__ + "."))
         res = set()
         for s in submodules:
-            with suppress(Exception):
```

Comment:
what does pyright not like here? i'd argue that in this case with-suppress is more readable

Reply:
If the exception is raised ms might be undefined in the original code and the for loop below would raise. Could probably also fix this by letting ms be defined to [] before this block